website/src/routes/text-field/+page.svelte

```diff
@@ -0,0 +1,140 @@
+            placeholder="placeholder"
+            variant="outlined"
+            color="primary"
+            helperText="helper text"
+            characterCounter={true}
+            maxlength={60}
+          />
+          <TextField
+            label="color & onSurfaceColor"
+            variant="outlined"
+            color="secondary"
+            onSurfaceColor="primary"
+            helperText="helper text"
+          />
+          <TextField disabled label="input disabled" variant="outlined" color="primary" />
+          <TextField label="1 icon" variant="outlined" color="primary" maxlength={60}>
+            <TextFieldIcon slot="leadingIcon" class="material-symbols-outlined"
+              >favorite</TextFieldIcon
+            >
+          </TextField>
+          <TextField label="2 icons" variant="outlined" color="primary">
+            <TextFieldIcon slot="leadingIcon" class="material-symbols-outlined">favorite</TextFieldIcon>
+            <TextFieldIcon slot="trailingIcon" class="material-symbols-outlined">bolt</TextFieldIcon>
+          </TextField>
+          <form>
+            <TextField
+              required
+              variant="outlined"
+              label="required"
+              pattern=".+@.+\\.com"
+              helperText="try email format"
+              placeholder="try submitting"
+              maxlength={20}
+              minlength={8}
+            />
+            <button style={'margin-top: 2px;'} type="submit">Submit</button>
+          </form>     
+          `}
+        </pre>
+        <Box class="text-field-example">
+          <TextField
+            label="my label"
+            variant="outlined"
+            color="primary"
+            helperText="helper text"
+            characterCounter={true}
+          />
+          <TextField
+            label="my label"
+            placeholder="placeholder"
+            variant="outlined"
+            color="primary"
+            helperText="helper text"
+            characterCounter={true}
+            maxlength={60}
+          />
+          <TextField
+            label="color & onSurfaceColor"
+            variant="outlined"
+            color="secondary"
+            onSurfaceColor="primary"
+            helperText="helper text"
+          />
+          <TextField disabled label="input disabled" variant="outlined" color="primary" />
+          <TextField label="1 icon" variant="outlined" color="primary" maxlength={60}>
+            <TextFieldIcon slot="leadingIcon" class="material-symbols-outlined"
+              >favorite</TextFieldIcon
+            >
+          </TextField>
+          <TextField label="2 icons" variant="outlined" color="primary">
+            <TextFieldIcon slot="leadingIcon" class="material-symbols-outlined"
+              >favorite</TextFieldIcon
+            >
+            <TextFieldIcon slot="trailingIcon" class="material-symbols-outlined">bolt</TextFieldIcon
+            >
+          </TextField>
+          <form>
```

Comment:
Can we extract this into a "Example: Validation" accordion and use a UnicornUI button instead of native.

Reply:
yeah I'll do that.